src/pcapi/core/offers/api.py

```diff
@@ -282,10 +333,92 @@ def as_utc_without_timezone(d: datetime.datetime) -> datetime.datetime:
 
     if feature_queries.is_active(FeatureToggle.SYNCHRONIZE_ALGOLIA):
         redis.add_offer_id(client=app.redis_client, offer_id=stock.offerId)
-
     return stock
 
 
+def bulk_create_edit_stocks(stockDataList: List[Union[StockCreationBodyModel, StockEditionBodyModel]]):
```

Comment:
```suggestion
def bulk_create_edit_stocks(stock_data_list: List[Union[StockCreationBodyModel, StockEditionBodyModel]]):
```

C'est fatiguant de passer de la convention de nommage JavaScript à celle de Python, hein ? :) J'ai le problème inverse quand j'écris du JavaScript !